src/queries/options.ts

```diff
@@ -12,3 +13,7 @@ export type TextMatchOptions = {
   exact?: boolean;
   normalizer?: NormalizerFn;
 };
+
+export const shouldReturnCompositeComponent = () => {
```

Comment:
The naming is a bit confusing, as it might suggest that this fuction is relevant to all queries. While in reality it only applied to some.

Instead of that I would suggest adding following to relevant queries (byText, ...):
`const shouldReturnHostElement = getConfig().useBreakingChanges`

Reply:
Definitely agree, I made the change